packages/react-dev-utils/failFast.js

```diff
@@ -0,0 +1,746 @@
+  'margin-top': '-3em',
+  'margin-bottom': '3em',
+  'text-align': 'right',
+  color: darkGray
+};
+
+var hintStyle = {
+  padding: '0.5em 1em',
+  cursor: 'pointer'
+};
+
+var closeButtonStyle = {
+  'font-size': '26px',
+  color: black,
+  padding: '0.5em 1em',
+  cursor: 'pointer',
+  position: 'absolute',
+  right: 0,
+  top: 0
+};
+
+var additionalStyle = {
+  'margin-bottom': '1.5em',
+  'margin-top': '-4em'
+};
+
+var headerStyle = {
+  'font-size': '1.7em',
+  'font-weight': 'bold',
+  color: red
+};
+
+var functionNameStyle = {
+  'margin-top': '1em',
+  'font-size': '1.2em'
+};
+
+var linkStyle = {
+  'font-size': '0.9em'
+};
+
+var anchorStyle = {
+  'text-decoration': 'none',
+  color: darkGray
+};
+
+var traceStyle = {
+  'font-size': '1em'
+};
+
+var depStyle = {
+  'font-size': '1.2em'
+};
+
+var primaryErrorStyle = {
+  'background-color': lightRed
+};
+
+var secondaryErrorStyle = {
+  'background-color': yellow
+};
+
+var omittedFramesStyle = {
+  color: black,
+  'font-size': '0.9em',
+  'margin': '1.5em 0',
+  cursor: 'pointer'
+};
+
+var preStyle = {
+  display: 'block',
+  padding: '0.5em',
+  'margin-top': '1.5em',
+  'margin-bottom': '0px',
+  'overflow-x': 'auto',
+  'font-size': '1.1em',
+  'white-space': 'pre'
+};
+
+var toggleStyle = {
+  'margin-bottom': '1.5em',
+  color: darkGray,
+  cursor: 'pointer'
+};
+
+var codeStyle = {
+  'font-family': 'Consolas, Menlo, monospace'
+};
+
+var hiddenStyle = {
+  display: 'none'
+};
+
+var groupStyle = {
+  'margin-left': '1em'
+};
+
+var _groupElemStyle = {
+  'background-color': 'inherit',
+  'border-color': '#ddd',
+  'border-width': '1px',
+  'border-radius': '4px',
+  'border-style': 'solid',
+  padding: '3px 6px',
+  cursor: 'pointer'
+};
+
+var groupElemLeft = Object.assign({}, _groupElemStyle, {
+  'border-top-right-radius': '0px',
+  'border-bottom-right-radius': '0px',
+  'margin-right': '0px'
+});
+
+var groupElemRight = Object.assign({}, _groupElemStyle, {
+  'border-top-left-radius': '0px',
+  'border-bottom-left-radius': '0px',
+  'margin-left': '-1px'
+});
+
+var footerStyle = {
+  'text-align': 'center',
+  color: darkGray
+};
+
+function applyStyles(element, styles) {
+  element.setAttribute('style', '');
+  // Firefox can't handle const due to non-compliant implementation
+  // Revisit Jan 2016
+  // https://developer.mozilla.org/en-US/Firefox/Releases/51#JavaScript
+  // https://bugzilla.mozilla.org/show_bug.cgi?id=1101653
+  for (var key in styles) {
+    if (!styles.hasOwnProperty(key)) continue;
+    var val = styles[key];
+    if (typeof val === 'function') val = val();
+    element.style[key] = val.toString();
+  }
+}
+
+var overlayReference = null;
+var additionalReference = null;
+var capturedErrors = [];
+var viewIndex = -1;
+var frameSettings = [];
+
+function consumeEvent(e) {
+  e.preventDefault();
+  e.target.blur();
+}
+
+function accessify(node) {
+  node.setAttribute('tabindex', 0);
+  node.addEventListener('keydown', function (e) {
```

Comment:
Ya it was more about memory leaks but Dan's point is valid, its only development. Carry on :)